parsers/jscript.c

```diff
@@ -511,11 +511,15 @@ static void parseTemplateString (vString *const string)
 		c = getcFromInputFile ();
 		if (c == '`')
 			break;
-		vStringPut (string, c);
+
+		if (c != EOF)
```

Comment:
should rather be in the previous `if` clause (even though it kind of makes the loop condition useless, but still):
```c
		if (c == '`' || c == EOF)
			break;
```

Reply:
Thank you. Updated.